docs/navigation-configuration.md

```diff
@@ -1,242 +1,242 @@
+### pathSegment
+This is the path in the browser URL. The main application path is built from values in the navigation path, joined with the **/** character. For example, if the value of this parameter is `home`, the path will be `yourwebsite.com/home`. You can override this setting by using one of the following instead of **pathSegment**: 
+* **link** - define a specific internal path. Note that a valid path must always start from the **root node**. For example, if your root node is `home`, and you want to navigate to the `projects` directory:
+	- `link: '/home/projects'` is correct
+	- `link: '/projects'`is not correct, since `/projects` is not the root node 
+* **externalLink** - takes you to an external website. You must define the **url** of the website and whether it should be opened in a new window (`sameWindow: false`) or the same window (`sameWindow: true`).
+### label
+The name of the node which will be visible on your page.
+### viewUrl
+The URL of the micro frontend which will be displayed in the main content area of your page. 
 
-````
-LuigiClient.linkManager().fromContext('project').withParam({sort: 'asc'}).navigate('/products');
-````
+## Grouping navigation nodes
 
-You can also navigate directly from any other node:
+You may use these parameters if you want to group related navigation nodes:
 
-````
-LuigiClient.linkManager().withParam({sort: 'asc'}).navigate('/something/sample_1/products');
-````
+### category 
+Simply add the **category** property to the navigation nodes you want to group, and they will be rendered in a dropdown. You should define at least one node in a group with **label** and **icon** properties. For all other nodes, you can set category as a string with the label value. The **collapsible** property is optional and defines whether the dropdown is expanded by default or not. 
 
-## Application path
+```javascript
+{
+  category: { label: 'Links', icon: 'myIcon', collapsible: true },
+  externalLink: {
+    url: 'http://www.google.com',
+    sameWindow: false
+  },
+  label: 'Click here to visit Google.com',
+}, 
+...
+ ```
 
-The main application path is built from **pathSegment** values in the navigation path, joined with the **/** character. You can override this setting using either **externalLink** or **link** parameters.
+### viewGroup
 
-The micro frontend view URL is the value of the **viewUrl** property of the last node in the navigation path.
+Imagine your application hosts two micro frontend views: `http://mysite.com/a#e` and  `http://mysite.com/b#f`. Due to hash routing and a different path up to `#`, they are, by default, rendered in different iframes. However, as they both have the **same origin**, such as`mysite.com`, and belong to the **same micro frontend** you want to render them in the same iframe. To achieve that, use the view groups feature. Define the **viewGroup** parameter for top navigation nodes. The children nodes will automatically be considered as part of the same view group. 
```

Comment:
"Define the **viewGroup** parameter for top navigation nodes." - this might be confusing as the term 'top navigation' is used to refer to the hotizontal top navigation area. You can define viewGroup on a node on any level belonging to top or left navigation.